### ISSUE TITLE
Unable to find image 'sentry-self-hosted-jq-local:latest' while installing 22.11.0

### ISSUE THREAD

**Issue by thelotusflower** [getsentry/self-hosted#1819]
### Self-Hosted Version

22.11.0

### CPU Architecture

x86_64

### Docker Version

20.10.9

### Docker Compose Version

2.6.0

### Steps to Reproduce

1. Download tar.gz with latest release https://github.com/getsentry/self-hosted/releases/tag/22.11.0
2. Unpack it 
3. Run ./install.sh from unpacked directory

### Expected Result

Self-hosted sentry installation successfully finishes 

### Actual Result

```
failed to solve: executor failed running [/bin/sh -c apt-get update && apt-get install -y --no-install-recommends cron &&     rm -r /var/lib/apt/lists/*]: exit code: 100
Unable to find image 'sentry-self-hosted-jq-local:latest' locally
Unable to find image 'sentry-self-hosted-jq-local:latest' locally
docker: Error response from daemon: pull access denied for sentry-self-hosted-jq-local, repository does not exist or may require 'docker login': denied: requested access to the resource is denied.
See 'docker run --help'.
docker: Error response from daemon: pull access denied for sentry-self-hosted-jq-local, repository does not exist or may require 'docker login': denied: requested access to the resource is denied.
See 'docker run --help'.
```

**Comment by hubertdeng123** [comment]
I'm wondering if the `sentry-self-hosted-jq-local` isn't built properly. That should be done in `build-docker-images.sh`. Can you confirm that is the case? I've just performed a fresh install with 22.11.0 locally and it seemed to work

**Comment by algethamy** [comment]
I got the same issue
this is the entire log:
```
▶ Parsing command line ...

▶ Detecting Docker platform
Detected Docker platform is linux/amd64

▶ Initializing Docker Compose ...

▶ Setting up error handling ...

Hey, so ... we would love to automatically find out about issues with your
Sentry instance so that we can improve the product. Turns out there is an app
for that, called Sentry. Would you be willing to let us automatically send data
about your instance upstream to Sentry for development and debugging purposes?

  y / yes / 1
  n / no / 0

(Btw, we send this to our own self-hosted Sentry instance, not to Sentry SaaS,
so that we can be in this together.)

Here's the info we may collect:

  - OS username
  - IP address
  - install log
  - runtime errors
  - performance data

Thirty (30) day retention. No marketing. Privacy policy at sentry.io/privacy.

y or n?
Understood. To avoid this prompt in the future, use one of these flags:

  --report-self-hosted-issues
  --no-report-self-hosted-issues

or set the REPORT_SELF_HOSTED_ISSUES environment variable:

  REPORT_SELF_HOSTED_ISSUES=1 to send data
  REPORT_SELF_HOSTED_ISSUES=0 to not send data


▶ Checking for latest commit ...

▶ Checking minimum requirements ...
Found Docker version 20.10.17
Found Docker Compose version 2.5.0
Unable to find image 'busybox:latest' locally
latest: Pulling from library/busybox
405fecb6a2fa: Pulling fs layer
405fecb6a2fa: Verifying Checksum
405fecb6a2fa: Download complete
405fecb6a2fa: Pull complete
Digest: sha256:fcd85228d7a25feb59f101ac3a955d27c80df4ad824d65f5757a954831450185
Status: Downloaded newer image for busybox:latest

▶ Turning things off ...
sentry-self-hosted  Warning: No resource found to remove

▶ Creating volumes for persistent storage ...
Created sentry-clickhouse.
Created sentry-data.
Created sentry-kafka.
Created sentry-postgres.
Created sentry-redis.
Created sentry-symbolicator.
Created sentry-zookeeper.

▶ Ensuring files from examples ...
../sentry/sentry.conf.py already exists, skipped creation.
../sentry/config.yml already exists, skipped creation.
../symbolicator/config.yml already exists, skipped creation.

▶ Ensuring Relay credentials ...
../relay/config.yml already exists, skipped creation.
../relay/credentials.json already exists, skipped creation.

▶ Generating secret key ...

▶ Fetching and updating Docker images ...
nightly: Pulling from getsentry/sentry
7a6db449b51b: Already exists
e238bceb2957: Already exists
9c37eda4ff12: Already exists
120078faa760: Already exists
2d3523ccd6f4: Already exists
84795dc9a9e4: Pulling fs layer
ade853742423: Pulling fs layer
7501b20fea5d: Pulling fs layer
4c81a7b67ef0: Pulling fs layer
4feb8d0d36fb: Pulling fs layer
fb7b2a95a184: Pulling fs layer
8048ec4739c0: Pulling fs layer
0056df2672f3: Pulling fs layer
0b907cf0abcc: Pulling fs layer
32f6d148e31c: Pulling fs layer
4c81a7b67ef0: Waiting
4feb8d0d36fb: Waiting
fb7b2a95a184: Waiting
8048ec4739c0: Waiting
0056df2672f3: Waiting
0b907cf0abcc: Waiting
32f6d148e31c: Waiting
7501b20fea5d: Download complete
84795dc9a9e4: Verifying Checksum
84795dc9a9e4: Download complete
84795dc9a9e4: Pull complete
ade853742423: Verifying Checksum
ade853742423: Pull complete
4feb8d0d36fb: Verifying Checksum
4feb8d0d36fb: Download complete
7501b20fea5d: Pull complete
8048ec4739c0: Verifying Checksum
8048ec4739c0: Download complete
fb7b2a95a184: Verifying Checksum
fb7b2a95a184: Download complete
0056df2672f3: Download complete
0b907cf0abcc: Verifying Checksum
0b907cf0abcc: Download complete
32f6d148e31c: Verifying Checksum
32f6d148e31c: Download complete
4c81a7b67ef0: Verifying Checksum
4c81a7b67ef0: Download complete
4c81a7b67ef0: Pull complete
4feb8d0d36fb: Pull complete
fb7b2a95a184: Pull complete
8048ec4739c0: Pull complete
0056df2672f3: Pull complete
0b907cf0abcc: Pull complete
32f6d148e31c: Pull complete
Digest: sha256:1df78772495e68a6283796626f870e7cf5e0c295b66a7941dfb2385d339b8f1b
Status: Downloaded newer image for getsentry/sentry:nightly
docker.io/getsentry/sentry:nightly

▶ Building and tagging Docker images ...

#1 [internal] load build definition from Dockerfile
#1 transferring dockerfile: 463B done
#1 DONE 0.0s

#2 [internal] load .dockerignore
#2 transferring context: 2B done
#2 DONE 0.0s

#3 [internal] load metadata for docker.io/getsentry/sentry:nightly
#3 DONE 0.0s

#4 [internal] load build context
#4 transferring context: 25.12kB done
#4 DONE 0.0s

#5 [1/4] FROM docker.io/getsentry/sentry:nightly
#5 DONE 0.3s

#6 [2/4] COPY . /usr/src/sentry
#6 DONE 20.4s

#7 [3/4] RUN if [ -s /usr/src/sentry/enhance-image.sh ]; then     /usr/src/sentry/enhance-image.sh; fi
#7 DONE 0.3s

#8 [4/4] RUN if [ -s /usr/src/sentry/requirements.txt ]; then     echo "sentry/requirements.txt is deprecated, use sentry/enhance-image.sh - see https://github.com/getsentry/self-hosted#enhance-sentry-image";     pip install -r /usr/src/sentry/requirements.txt; fi
#8 DONE 0.5s

#9 exporting to image
#9 exporting layers 0.0s done
#9 writing image sha256:b195113f1fa54476a6b65bea12a076569926a251c78613be2085b1d77f087b8c done
#9 naming to docker.io/library/sentry-self-hosted-local done
#9 DONE 0.1s
#1 [sentry-self-hosted-local internal] load build definition from Dockerfile
#1 transferring dockerfile: 32B done
#1 DONE 0.0s

#2 [sentry-cleanup-self-hosted-local internal] load build definition from Dockerfile
#2 transferring dockerfile: 469B done
#2 DONE 0.0s

#3 [snuba-cleanup-self-hosted-local internal] load build definition from Dockerfile
#3 transferring dockerfile: 469B done
#3 DONE 0.0s

#4 [symbolicator-cleanup-self-hosted-local internal] load build definition from Dockerfile
#4 transferring dockerfile: 469B done
#4 DONE 0.0s

#5 [clickhouse-self-hosted-local internal] load build definition from Dockerfile
#5 transferring dockerfile: 71B done
#5 DONE 0.0s

#6 [sentry-cleanup-self-hosted-local internal] load .dockerignore
#6 transferring context: 2B done
#6 DONE 0.0s

#7 [sentry-self-hosted-local internal] load .dockerignore
#7 transferring context: 2B done
#7 DONE 0.0s

#8 [snuba-cleanup-self-hosted-local internal] load .dockerignore
#8 transferring context: 2B done
#8 DONE 0.0s

#9 [symbolicator-cleanup-self-hosted-local internal] load .dockerignore
#9 transferring context: 2B done
#9 DONE 0.0s

#10 [clickhouse-self-hosted-local internal] load .dockerignore
#10 transferring context: 2B done
#10 DONE 0.0s

#11 [sentry-cleanup-self-hosted-local internal] load metadata for docker.io/library/sentry-self-hosted-local:latest
#11 DONE 0.0s

#12 [sentry-self-hosted-local internal] load metadata for docker.io/getsentry/sentry:nightly
#12 DONE 0.0s

#13 [sentry-self-hosted-local 1/4] FROM docker.io/getsentry/sentry:nightly
#13 DONE 0.0s

#14 [sentry-self-hosted-local internal] load build context
#14 transferring context: 297B done
#14 DONE 0.0s

#15 [sentry-self-hosted-local 2/4] COPY . /usr/src/sentry
#15 CACHED

#16 [sentry-self-hosted-local 3/4] RUN if [ -s /usr/src/sentry/enhance-image.sh ]; then     /usr/src/sentry/enhance-image.sh; fi
#16 CACHED

#17 [sentry-self-hosted-local 4/4] RUN if [ -s /usr/src/sentry/requirements.txt ]; then     echo "sentry/requirements.txt is deprecated, use sentry/enhance-image.sh - see https://github.com/getsentry/self-hosted#enhance-sentry-image";     pip install -r /usr/src/sentry/requirements.txt; fi
#17 CACHED

#18 [sentry-self-hosted-local] exporting to image
#18 exporting layers done
#18 writing image sha256:b195113f1fa54476a6b65bea12a076569926a251c78613be2085b1d77f087b8c done
#18 naming to docker.io/library/sentry-self-hosted-local done
#18 DONE 0.0s

#19 [sentry-cleanup-self-hosted-local internal] load build context
#19 transferring context: 658B done
#19 DONE 0.0s

#20 [symbolicator-cleanup-self-hosted-local internal] load metadata for docker.io/getsentry/symbolicator:nightly
#20 DONE 0.0s

#21 [snuba-cleanup-self-hosted-local internal] load metadata for docker.io/getsentry/snuba:nightly
#21 DONE 0.0s

#22 [snuba-cleanup-self-hosted-local internal] load build context
#22 transferring context: 658B done
#22 DONE 0.0s

#23 [symbolicator-cleanup-self-hosted-local internal] load build context
#23 transferring context: 658B done
#23 DONE 0.1s

#24 [snuba-cleanup-self-hosted-local 1/5] FROM docker.io/getsentry/snuba:nightly
#24 DONE 0.3s

#25 [symbolicator-cleanup-self-hosted-local 1/5] FROM docker.io/getsentry/symbolicator:nightly
#25 DONE 0.2s

#26 [symbolicator-cleanup-self-hosted-local 2/5] RUN if [ -z "${http_proxy}" ]; then echo "Acquire::http::proxy "${http_proxy}";" >> /etc/apt/apt.conf; fi
#26 ...

#27 [sentry-cleanup-self-hosted-local 1/5] FROM docker.io/library/sentry-self-hosted-local
#27 DONE 0.4s

#28 [sentry-cleanup-self-hosted-local 2/5] RUN if [ -z "${http_proxy}" ]; then echo "Acquire::http::proxy "${http_proxy}";" >> /etc/apt/apt.conf; fi
#28 ...

#26 [symbolicator-cleanup-self-hosted-local 2/5] RUN if [ -z "${http_proxy}" ]; then echo "Acquire::http::proxy "${http_proxy}";" >> /etc/apt/apt.conf; fi
#26 DONE 0.4s

#29 [snuba-cleanup-self-hosted-local 2/5] RUN if [ -z "${http_proxy}" ]; then echo "Acquire::http::proxy "${http_proxy}";" >> /etc/apt/apt.conf; fi
#29 DONE 0.4s

#30 [snuba-cleanup-self-hosted-local 3/5] RUN if [ -z "${https_proxy}" ]; then echo "Acquire::https::proxy "${https_proxy}";" >> /etc/apt/apt.conf; fi
#30 ...

#28 [sentry-cleanup-self-hosted-local 2/5] RUN if [ -z "${http_proxy}" ]; then echo "Acquire::http::proxy "${http_proxy}";" >> /etc/apt/apt.conf; fi
#28 DONE 0.6s

#31 [symbolicator-cleanup-self-hosted-local 3/5] RUN if [ -z "${https_proxy}" ]; then echo "Acquire::https::proxy "${https_proxy}";" >> /etc/apt/apt.conf; fi
#31 DONE 0.6s

#32 [sentry-cleanup-self-hosted-local 3/5] RUN if [ -z "${https_proxy}" ]; then echo "Acquire::https::proxy "${https_proxy}";" >> /etc/apt/apt.conf; fi
#32 ...

#30 [snuba-cleanup-self-hosted-local 3/5] RUN if [ -z "${https_proxy}" ]; then echo "Acquire::https::proxy "${https_proxy}";" >> /etc/apt/apt.conf; fi
#30 DONE 0.6s

#33 [snuba-cleanup-self-hosted-local 4/5] RUN apt-get update && apt-get install -y --no-install-recommends cron &&     rm -r /var/lib/apt/lists/*
#33 ...

#32 [sentry-cleanup-self-hosted-local 3/5] RUN if [ -z "${https_proxy}" ]; then echo "Acquire::https::proxy "${https_proxy}";" >> /etc/apt/apt.conf; fi
#32 DONE 0.7s

#34 [sentry-cleanup-self-hosted-local 4/5] RUN apt-get update && apt-get install -y --no-install-recommends cron &&     rm -r /var/lib/apt/lists/*
#34 0.540 Err:1 http://deb.debian.org/debian bullseye InRelease
#34 0.540   Temporary failure resolving 'deb.debian.org'
#34 CANCELED

#35 [clickhouse-self-hosted-local internal] load metadata for docker.io/yandex/clickhouse-server:20.3.9.70
#35 ERROR: failed to authorize: rpc error: code = Unavailable desc = transport is closing

#36 [symbolicator-cleanup-self-hosted-local 4/5] RUN apt-get update && apt-get install -y --no-install-recommends cron &&     rm -r /var/lib/apt/lists/*
#0 0.670 Err:1 http://deb.debian.org/debian bullseye InRelease
#0 0.670   Temporary failure resolving 'deb.debian.org'
#0 5.675 Err:2 http://deb.debian.org/debian-security bullseye-security InRelease
#0 5.675   Temporary failure resolving 'deb.debian.org'
#0 5.675 Err:3 http://deb.debian.org/debian bullseye-updates InRelease
#0 5.675   Temporary failure resolving 'deb.debian.org'
#0 5.678 Reading package lists...
#0 5.687 W: Failed to fetch http://deb.debian.org/debian/dists/bullseye/InRelease  Temporary failure resolving 'deb.debian.org'
#0 5.687 W: Failed to fetch http://deb.debian.org/debian-security/dists/bullseye-security/InRelease  Temporary failure resolving 'deb.debian.org'
#0 5.687 W: Failed to fetch http://deb.debian.org/debian/dists/bullseye-updates/InRelease  Temporary failure resolving 'deb.debian.org'
#0 5.687 W: Some index files failed to download. They have been ignored, or old ones used instead.
#0 5.695 Reading package lists...
#0 5.703 Building dependency tree...
#0 5.704 Reading state information...
#0 5.707 E: Unable to locate package cron
#36 ERROR: executor failed running [/bin/sh -c apt-get update && apt-get install -y --no-install-recommends cron &&     rm -r /var/lib/apt/lists/*]: exit code: 100

#33 [snuba-cleanup-self-hosted-local 4/5] RUN apt-get update && apt-get install -y --no-install-recommends cron &&     rm -r /var/lib/apt/lists/*
#33 0.877 Err:1 http://deb.debian.org/debian bullseye InRelease
#33 0.877   Temporary failure resolving 'deb.debian.org'
#33 CANCELED
------
 > [clickhouse-self-hosted-local internal] load metadata for docker.io/yandex/clickhouse-server:20.3.9.70:
------
------
 > [symbolicator-cleanup-self-hosted-local 4/5] RUN apt-get update && apt-get install -y --no-install-recommends cron &&     rm -r /var/lib/apt/lists/*:
#0 5.675   Temporary failure resolving 'deb.debian.org'
#0 5.678 Reading package lists...
#0 5.687 W: Failed to fetch http://deb.debian.org/debian/dists/bullseye/InRelease  Temporary failure resolving 'deb.debian.org'
#0 5.687 W: Failed to fetch http://deb.debian.org/debian-security/dists/bullseye-security/InRelease  Temporary failure resolving 'deb.debian.org'
#0 5.687 W: Failed to fetch http://deb.debian.org/debian/dists/bullseye-updates/InRelease  Temporary failure resolving 'deb.debian.org'
#0 5.687 W: Some index files failed to download. They have been ignored, or old ones used instead.
#0 5.695 Reading package lists...
#0 5.703 Building dependency tree...
#0 5.704 Reading state information...
#0 5.707 E: Unable to locate package cron
------
failed to solve: executor failed running [/bin/sh -c apt-get update && apt-get install -y --no-install-recommends cron &&     rm -r /var/lib/apt/lists/*]: exit code: 100
Unable to find image 'sentry-self-hosted-jq-local:latest' locally
Unable to find image 'sentry-self-hosted-jq-local:latest' locally
docker: Error response from daemon: pull access denied for sentry-self-hosted-jq-local, repository does not exist or may require 'docker login': denied: requested access to the resource is denied.
See 'docker run --help'.
docker: Error response from daemon: pull access denied for sentry-self-hosted-jq-local, repository does not exist or may require 'docker login': denied: requested access to the resource is denied.
See 'docker run --help'.
```

**Comment by sharokh1** [comment]
same issue here.. I also had something similar last time #1781 and it resolved itself after a while

**Comment by thelotusflower** [comment]
> I'm wondering if the `sentry-self-hosted-jq-local` isn't built properly. That should be done in `build-docker-images.sh`. Can you confirm that is the case? I've just performed a fresh install with 22.11.0 locally and it seemed to work

Yeah the problem is that docker can't resolve deb.debian.org during buld phase. I guess the problem is on my side.

**Comment by thelotusflower** [comment]
@algethamy you have the same issue as I had: **Temporary failure resolving 'deb.debian.org'**. In my case this was caused by the fact that my host machine uses ipv6 and resolv.conf had only ipv6 dns addresses. I resolved this by adding 8.8.8.8 and 8.8.4.4 in resolv.conf
